### DISCUSSION TITLE
Default chunk size too big for default embedding model?

### DISCUSSION THREAD

**Discussion by lbluett** [open-webui/open-webui#6212] [General]
Hi all, this may be due to my own ignorance but isn't the default chunk size too big for the default embedding model?

The default embedding model is: sentence-transformers/all-MiniLM-L6-v2
The default chunk size is: 1500

Isn't the max sequence length for all-MiniLM-L6-v2 256 tokens? According to: https://www.sbert.net/docs/sentence_transformer/pretrained_models.html

I believe this would result in the 1500 token chunks being truncuated to 256 tokens which means a lot of context is dropped.

**Comment by thiswillbeyourgithub** [comment]
I brought it up multiple times and got ignored each time. See #5323 

I think it's counting letters instead of tokens, which is complete bonkers IMO

**Comment by tjbck** [comment]
Token based text splitting has been added to dev, also @thiswillbeyourgithub your posts are not being ignored. As you're very well aware by now, I'm the only one working on the project and no one else. Without any exaggeration, I'm getting hundreds emails from people everyday + I have a full-time job of my own because this project alone does not pay the bills. 


I appreciate your enthusiasm regarding the project but I don't appreciate how on every post you're claiming it's not being looked at and I'm somehow actively ignoring the "issues", and treating us as if we have a group of engineers at disposal, which is certainly not the case. There are things I'll be prioritising because some of them have been backlogged for months, and some will have to wait. I've acknowledged and moved your posts to discussions section, hoping people in the community could contribute. With that being said, I'll slowly get to them one by one as I have been, so I would really appreciate your patience here, as I really don't want to burn myself (or anyone for that matter) out. Thanks for your understanding!

**Reply by thiswillbeyourgithub** [reply]
I’m truly sorry for my previous comments. I’m still incredibly enthusiastic about the project and use it extensively, both for myself and with friends. I realize now that I got carried away, overestimating the resources available and not knowing you were handling this alone alongside a full-time job. Your approach to avoiding burnout is commendable, and I deeply appreciate everything you’ve done and continue to do for the project. Thank you for your patience and hard work. Take care and have a nice day.

**Comment by thistleknot** [comment]
oh snap, this project is great, sorry to hear it's a 1 man show.  I'd like to help.  I suppose I just need to start creating some pr's.
